src/main/java/com/spotinst/sdkjava/model/SpotinstRepoManager.java

```diff
@@ -156,5 +159,7 @@ public void setSpotinstElastigroupEventLogRepo(ISpotinstEventsLogsRepo spotinstE
         this.spotinstElastigroupEventLogRepo = spotinstElastigroupEventLogRepo;
     }
     //endregion
-
+    public ISpotinstOceanGKECreateLaunchSpecRepo getSpotinstOceanGKECreateLaunchSpecRepoGcp() {
```

Comment:
This class should have a set function also.

Reply:
As this only returns the specific class object, So set method is not required it seems